environment.yml

```diff
@@ -1,10 +1,10 @@
-name: <Dashboard name>
+name: plotly_dashboard_template # Replace with your dashboard name
```

Comment:
SUSPICIOUS.

This has been reported.

![](https://i.stack.imgur.com/sm7og.png)